From c07bfa4b4eae0640096ac7be5378d59272d54b1e Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 13 Dec 2023 14:35:17 +0200
Subject: [PATCH] [SW-0] gaudi3: copy entire pci phy fw

PO Tests show that we need to copy the entire FW of the PHY.
Later we can optimize this by writing trailing zeroes
synthetically instead of storing them in flash and copying
them from flash.

Change-Id: I28261d98bf33ef5880b4c4adc586e2d16d49f649
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/common/pcie_phy_fw.mk b/arch/gaudi3/common/pcie_phy_fw.mk
index 2a0e729..1c284da 100644
--- a/arch/gaudi3/common/pcie_phy_fw.mk
+++ b/arch/gaudi3/common/pcie_phy_fw.mk
@@ -1 +1,3 @@
-PHY_FW_SIZE := 0xA500
\ No newline at end of file
+# PHY_FW_SIZE := 0xA500
+# TODO: optimize phy fw size, do not copy trailing zeroes
+PHY_FW_SIZE := 0x10000
\ No newline at end of file
